MAINTAINERS.yml

```diff
@@ -2724,10 +2724,12 @@ Networking:
     - pdgendt
     - tbursztyka
     - ssharks
+    - mustafaabdullahk
```

Comment:
I am not sure this is a good idea, now you will be start to get review requests for all the network PRs.
Probably better to create a separate Prometheus section and mark you as a maintainer there.

Reply:
Updated with your suggestions. Created Prometheus section under networking and marked as far as i understand.